CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ and this project adheres Fto [Semantic Versioning](http://semver.org/spec/v2.0.0
 - nghttp2 is now picked up from the OS rather than vendored to enable libcurl usage
 - Misc dependency updates (#6725)
 
+### Added
+
+- Joining nodes can now request a snapshot from their peers at startup, rather than relying on file access. The joinee's snapshot will be fetched and used if it is more recent than the joiner has access to.
```

Comment:
```suggestion
- Joining nodes can now request a snapshot from their peers at startup, rather than relying on file access. The joinee's snapshot will be fetched and used if it is more recent than the joiner has access to (#6758).
```